mbofui/app/js/utils.js

```diff
@@ -123,6 +123,16 @@ webshim.setOptions("forms-ext", {
   }
 });
 
+var checkTimeSlice  = function(start,end,currentView) {
+  if(moment(start).isAfter(moment()) && currentView ==='/api/events/current/'){
+    return {'type':'alert-success','message':'Event created for the future - view in upcoming events.'};
+  } else if (moment(start).isSameOrBefore(moment(),'hour') && moment(end).isSameOrAfter(moment(), 'hour')  && currentView ==='/api/events/upcoming/'){
+    return {'type':'alert-success','message':'Event created for the current - view in current events.'};
+  } else {
+    return null;
```

Comment:
Like we discussed - it is not an error - just a flag that the conditions were not met. The view knows what to do with it - which is namely nothing. There might be a better way doing this, like declaring it null in the controller before checking the conditions.